### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by pradepta-K**
Added list to Readme

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1 +1,3 @@
 # Heading
+
+This is the added list.
```
Comment (pradepta-K) [inline README]: What does this mean?

Comment (PradeptaK) [reply README]: idek

File: index.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<h1>This is a heading</h1>
```
Comment (PradeptaK) [inline index.html:1]: Can you add more?

Comment (PradeptaK) [reply index.html:1]: Yes I will add.

**Review by PradeptaK — COMMENTED** [review]
Yes I have reviewed but haven't given confirmation

**Review by PradeptaK — APPROVED** [review]
I approve

**Pull request merged**